File: Array Snacks/question4.py
def numbers_on_odd_positions(list):
	odd_index_numbers = []
	for index in range(0, len(list)):
		if index % 2 == 1:
			odd_index_numbers.append(list[index])
	return odd_index_numbers

print(numbers_on_odd_positions([1, 2, 3, 4, 5, 6, 7, 8, 9]))

File: Array Snacks/Question4.java
public class Question4{
   public int[] numbersOnOddPositions(int[] list){
	
	int[] oddIndexNumbers = new int[(list.length) / 2];

	int counter = 0;
	for(int index = 0; index < list.length; index++){
		if(index % 2 == 1){
			oddIndexNumbers[counter] = list[index];
			counter = counter + 1;
		}
	} 
	return oddIndexNumbers;
   }
}